ui/core/src/ui-core/contextmenu/ContextSubMenu.tsx

```diff
@@ -116,7 +116,6 @@ export class ContextSubMenu extends React.Component<ContextSubMenuProps, Context
           ref={(el) => { this._menuElement = el; }}
           className="core-context-submenu-popup"
           opened={this.state.opened}
-          selectedIndex={0}
           direction={renderDirection}
           parentSubmenu={this}
           {...contextMenuProps} >

```

Comment:
@DanEastBentley @bsteinbk TypeScript was complaining at line 119 here:

> 'selectedIndex' is specified more than once, so this usage will be overwritten.  **ts(2783)**
ContextSubMenu.tsx(122, 11): This spread always overwrites this property.

Does that mean it's safe to remove `selectedIndex` here?  Or do we need to fix the type of `contextMenuProps` somewhere?

Reply:
I think your change to delete it is the right thing to do.